docs/tutorials/02a_training_a_quantum_model_on_a_real_dataset.ipynb

```diff
@@ -0,0 +1,991 @@
+    "train_features, test_features, train_labels, test_labels = train_test_split(\n",
+    "    features, labels, train_size=0.8, random_state=algorithm_globals.random_seed\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "close-festival",
+   "metadata": {},
+   "source": [
+    "We train a classical Support Vector Classifier from scikit-learn. For the sake of simplicity, we don't tweak any parameters and rely on the default values."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "proved-reviewer",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from sklearn.svm import SVC\n",
+    "\n",
+    "svc = SVC()\n",
+    "_ = svc.fit(train_features, train_labels)  # suppress printing the return value"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "earned-destination",
+   "metadata": {},
+   "source": [
+    "Now we check out how well our classical model performs. We will analyze the scores in the conclusion section."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "veterinary-proxy",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Classical SVC on the training dataset: 0.99\n",
+      "Classical SVC on the test dataset:     0.97\n"
+     ]
+    }
+   ],
+   "source": [
+    "train_score_c4 = svc.score(train_features, train_labels)\n",
+    "test_score_c4 = svc.score(test_features, test_labels)\n",
+    "\n",
+    "print(f\"Classical SVC on the training dataset: {train_score_c4:.2f}\")\n",
+    "print(f\"Classical SVC on the test dataset:     {test_score_c4:.2f}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "limited-hybrid",
+   "metadata": {},
+   "source": [
+    "As can be seen from the scores, the classical SVC algorithm performs very well. Next up, it's time to look at quantum machine learning models.\n",
+    "\n",
+    "## 3. Training a Quantum Machine Learning Model\n",
+    "\n",
+    "As an example of a quantum model, we'll train a variational quantum classifier (VQC). The VQC is the simplest classifier available in Qiskit Machine Learning and is a good starting point for newcomers to quantum machine learning who have a background in classical machine learning.\n",
+    "\n",
+    "But before we train a model, let's examine what comprises the `VQC` class. Two of its central elements are the feature map and ansatz. What these are will now be explained.\n",
+    "\n",
+    "Our data is classical, meaning it consists of a set of bits, not qubits. We need a way to encode the data as qubits. This process is crucial if we want to obtain an effective quantum model. We usually refer to this mapping as data encoding, data embedding, or data loading. While feature mapping is a common ML mechanism, this process of loading data into quantum states does not appear in classical machine learning as that only operates in the classical world.\n",
```

Comment:
This and the following paragraph are explanations of two of the central elements it states. The second paragraph includes ansatz and I think thats clearer. The first has a reference to feature mapping being a common ML mechanism but then this data loading is just for quantum, so is it clear what the feature map is for here? I added extended one of the sentences thus `We usually refer to this mapping as data encoding, data embedding, or data loading and this is the role of the feature map.` in an attempt to tie things together better and make it clear what the feature map was for as it did not really seem to directly state that. I copied the paragraph here as its hard to see in this diff view!

So from: 
>Our data is classical, meaning it consists of a set of bits, not qubits. We need a way to encode the data as qubits. This process is crucial if we want to obtain an effective quantum model. We usually refer to this mapping as data encoding, data embedding, or data loading. While feature mapping is a common ML mechanism, this process of loading data into quantum states does not appear in classical machine learning as that only operates in the classical world.

To:
>Our data is classical, meaning it consists of a set of bits, not qubits. We need a way to encode the data as qubits. This process is crucial if we want to obtain an effective quantum model. We usually refer to this mapping as data encoding, data embedding, or data loading and this is the role of the feature map. While feature mapping is a common ML mechanism, this process of loading data into quantum states does not appear in classical machine learning as that only operates in the classical world.

Reply:
PS. I like the new seaborn plot!

Reply:
I see. Yep, that makes sense to me.

Reply:
@woodsp-ibm I have applied your suggestion.
P.S. seaborn is a nice library!